foundation-extensions/adobe-experience-platform-assurance/README.md

```diff
@@ -28,30 +28,12 @@ Follow these steps to add the install the extension in Experience Platform Launc
 ### Import the library to your app code
 
 {% tabs %}
-{% tab title="Android" %}
-**Java**
-
-1. Add the following libraries in your project's `build.gradle` file:
-
-   ```java
-   implementation 'com.adobe.marketing.mobile:core:1+'
-   implementation 'com.adobe.marketing.mobile:assurance:1+'
-   ```
-
-2. Import the Project Griffon libraries with the other SDK libraries:
-
-   ```java
-   import com.adobe.marketing.mobile.Assurance; 
-   import com.adobe.marketing.mobile.MobileCore;
-   ```
-{% endtab %}
-
 {% tab title="iOS" %}
 Add the library to your project via your [Cocoapods](https://cocoapods.org/pods/AEPAssurance) `Podfile`
 
 ```text
 pod 'ACPCore'
-pod 'AEPAssurance'
+pod 'AEPAssurance', '~> 1.0.0
```

Comment:
Based on https://guides.cocoapods.org/using/the-podfile.html I think this should be '~> 1.0' to include all the  versions until 2.x.